dv/uvm/core_ibex/yaml/rtl_simulation.yaml

```diff
@@ -110,6 +110,7 @@
   compile:
     cmd:
       - "qrun -f ibex_dv.f -uvmhome uvm-1.2
+              -svinputport=relaxed
```

Comment:
Can you try `-svinputport=net` here to see if the strict (LRM-compliant) mode works as well? I think it should, but it's easy to get lost in the SV spec.

Reply:
It has the same as result. So I changed it by `svinputport=net`